src/FixedDataTable.js

```diff
@@ -920,7 +920,7 @@ var FixedDataTable = React.createClass({
     }
 
     var lastScrollTop = oldState ? oldState.scrollTop : undefined;
-    if (props.scrollTop !== lastScrollTop) {
+    if (typeof props.scrollTop !== 'undefined' && props.scrollTop !== null && props.scrollTop !== lastScrollTop) {
```

Comment:
Not sure, I just copied it. I think we can just do props.scrollTop != null since undefined == null
Will update this and above